ext/pronto-v1.1/src/nvm_manager.hpp

```diff
@@ -42,7 +43,7 @@ class NVManager {
         void destroy(PersistentObject *);
 
         // Find pointer to persistent objects using its unique identifier
-        PersistentObject *findObject(string);
+        PersistentObject *findObject(std::string);
```

Comment:
But we already have `using namespace std` in this header file, right?

Reply:
Yeah I must have added this when I saw the error on `string` -- including `<string>` should be enough.